cli/injector/main.go

```diff
@@ -283,20 +274,24 @@ func initConfig() {
 			return
 		}
 
+		deadline, err = time.Parse(time.RFC3339, deadlineRaw)
+		if err != nil {
+			deadline = time.Now().Add(time.Hour)
```

Comment:
```suggestion
			deadline = time.Now().Add(time.Minute * 5)
```

If we can't determine, I'd rather fail to something safer?

Reply:
Sure, I did not wanted to change the actual behavior, however safer is better indeed.